pkg/common/grpcutil/log.go

```diff
@@ -0,0 +1,12 @@
+package grpcutil
+
+import (
+	"log"
+
+	"github.com/sirupsen/logrus"
+)
+
+func LoggerFromFieldLogger(fl logrus.FieldLogger) *log.Logger {
```

Comment:
nit: Should this return logrus.StdLogger instead?

Reply:
Not sure what's best :) updated anyways